bin/run_tests

```diff
@@ -24,8 +24,37 @@ PROJECT_DIR=$(cd $(dirname $(dirname $0)) && pwd)
 # Make sawtooth scripts accessible
 PATH=$PROJECT_DIR/bin:$PATH
 
+# Arguments to trigger the tests
+DIRECT_MODEL_TESTS=0
+PROXY_MODEL_TESTS=0
+
 main() {
-    run_docker_test ./ci/avalon-tests.yaml
+
+    if [ "$DIRECT_MODEL_TESTS" == "1" ]; then
+        run_docker_test ./ci/avalon-direct-model-tests.yaml
+    elif [ "$PROXY_MODEL_TESTS" == "1" ]; then
+        run_docker_test ./ci/avalon-proxy-model-tests.yaml
+    fi
 }
 
+while getopts "dph" opt; do
+    case $opt in
+        d)
+            DIRECT_MODEL_TESTS=1
+            ;;
+        p)
+            PROXY_MODEL_TESTS=1
+            ;;
+        h)
+            echo "Usage:"
```

Comment:
Before the `Usage:` line add a line explaining what `run_tests` does.  Something like this (reword or correct if needed):

`echo "Run Avalon CI tests in Docker"`

Reply:
Done